packages/p2p/webpack.config.js

```diff
@@ -131,15 +127,8 @@ module.exports = {
             react: 'react',
             'react-dom': 'react-dom',
             'prop-types': 'prop-types',
+            ...(is_publishing ? { formik: 'formik' } : {}),
```

Comment:
shouldn't this be `...(is_publishing ? {} : { formik: 'formik' }),` 🤔

Reply:
No, we want to pass `is_publishing` to the function regardless of its value, especially when generating the correct externals (as it'll return externals for when `is_publishing` is `false`, it'll return an empty object for `is_publishing` is `true`). For packages that are built as part of Deriv.app we know that Core will include these packages for us, for packages that are published we will have to resolve them.